lib/debootstrap.sh

```diff
@@ -562,9 +562,9 @@ PREPARE_IMAGE_SIZE
 		# Hardcoded overhead +25% is needed for desktop images,
 		# for CLI it could be lower. Align the size up to 4MiB
 		if [[ $BUILD_DESKTOP == yes ]]; then
-			local sdsize=$(bc -l <<< "scale=0; ((($imagesize * 1.30) / 1 + 0) / 4 + 1) * 4")
+			local sdsize=$(bc -l <<< "scale=0; ((($imagesize * 1.35) / 1 + 0) / 4 + 1) * 4")
```

Comment:
Shellcheck would report to remove the $.

Reply:
Don't understand what is wrong.

Reply:
Only `$(())` will report that. But here is `bc`